### DISCUSSION TITLE
Overlay Design Template

### DISCUSSION THREAD

**Discussion by audioop** [raphaelquast/beatstep#77] [General]
I've reached out to Daniel at Taktility with the overlay template provided here and he asked if I could send him one without all of the cutouts. could you please add a version of it without the cutouts to the repo? thanks!

**Comment by raphaelquast** [comment]
Hey, yep that's a good point!
I'll try to upload suitable versions of the overlay in the next few days!

**Comment by raphaelquast** [comment]
@audioop 
Hey, here are 3 hi-res versions for the overlay of BeatStep_Q v2.0: 

The cut-markers right now are such that you can put the overlay over the encoders **without having to remove the knobs first**.
If you want to have the overlay fixed by the knobs you'd need to mention it explicitly to David so that he will make the cut-outs for the encoders smaller.   (I hope it's clear what I mean by that)

[BeatStepQ_v2.0_overlay_only_cut_markers.pdf](https://github.com/raphaelquast/beatstep/files/7139825/BeatStepQ_v2.0_overlay_only_cut_markers.pdf)
[BeatStepQ_v2.0_overlay_with_cut_markers.pdf](https://github.com/raphaelquast/beatstep/files/7139826/BeatStepQ_v2.0_overlay_with_cut_markers.pdf)
[BeatStepQ_v2.0_overlay_without_cut_markers.pdf](https://github.com/raphaelquast/beatstep/files/7139827/BeatStepQ_v2.0_overlay_without_cut_markers.pdf)

Let me know if you need anything else!
Cheers, Raphael

**Reply by audioop** [reply]
thanks for the response. actually, I think that Daniel wants a template with none of the knob/pad shapes on it at all. he sent me your old template as an example

[SRA3_2236_Beatstep.pdf](https://github.com/raphaelquast/beatstep/files/7143680/SRA3_2236_Beatstep.pdf)

**Comment by raphaelquast** [comment]
aaah, OK 😄 
here you go:
[BeatStepQ_v2.0_overlay_no_pads_or_cut_markers.pdf](https://github.com/raphaelquast/beatstep/files/7143801/BeatStepQ_v2.0_overlay_no_pads_or_cut_markers.pdf)
